cmake/configs/nuttx_px4fmu-v2_default.cmake

```diff
@@ -46,6 +46,7 @@ set(config_module_list
 	drivers/pwm_input
 	drivers/camera_trigger
 	drivers/bst
+	drivers/snapdragon_rc_pwm
```

Comment:
Is there a specific benefit of using a separate new file over just running the MAVLink app as we do for offboard control anyway?